### DISCUSSION TITLE
【灌水】好和谐的绳网啊

### DISCUSSION THREAD

**Discussion by kankankeke114514** [share121/inter-knot#543] [其他]
新人报到！这就是绳网吗，感觉氛围好和谐！没有拉踩引战，即使有不同游戏也都是正常的交流，狠狠喜欢上绳网了！
![img](https://github.com/user-attachments/assets/efe2a56e-1c54-4b1b-83ef-d2b7433a3367)

**Comment by ssck1024** [comment]
毕竟要装插件的，小孩子可能不会

**Comment by FrostPhoenix79** [comment]
如果能发视频，我宣布这就是我未来的基地了

**Comment by capacitor1** [comment]
>Reply::如果能发视频，我宣布这就是我未来的基地了

目前GitHub上可以发些小的视频和文件，对文件体积有很大限制。。

但是如果能自己写代码来实现一下功能，那就更好了😋可以借此引用外部网站的文件和空间，发东西会方便些。

**Reply by FrostPhoenix79** [reply]
回楼上，我是没有办法了，小废物点心版绳匠一枚，只能求助广大绳匠了 (ㅇㅅㅇ❀)

**Comment by zzz14121** [comment]
毕竟相对于其他网站比较难进入 素质低的小孩子很难进来

**Comment by yiwangzj** [comment]
上绳网程序比较复杂，滤去了不是真心想要一个社区交流的人。

**Comment by trv3wood** [comment]
毕竟绳网是Github(划掉)Gayhub套壳[doge]，能上Github已经筛选了很多人

**Comment by rnfmabj-rnfmabj** [comment]
捕捉野生月计人（看我名字）

**Comment by shuyishaoxiancao** [comment]
那当然，那帮小鬼肯定没什么耐心，而且全是英文它们肯定看不懂

**Comment by Elysia-Ego-093** [comment]
十分赞同，github作为门槛确实可以筛选掉很多不明生物

**Comment by qiu-1214** [comment]
毕竟想要进绳网本身就有些难度，可以限制相当部分不明生物

**Comment by 1451374815** [comment]
上这个网站门槛比较高，而且没有商业价值。所以没有婴儿和水军，总体而言就只是一场太一之梦罢了，也不知道可以保持多久，可能热度一过就结束了

**Comment by llxcx4** [comment]
非常好的绳网ε٩(๑> ₃ <)۶з

**Comment by llxcx4** [comment]
非常好的绳网ε٩(๑> ₃ <)۶з

**Comment by GanlinSama** [comment]
希望可以靠着高门槛多撑一段时间吧……我还是挺看好的

**Comment by fy-sec** [comment]
非常好的绳网，爱来自新艾莉都

**Comment by Y-0930** [comment]


**Reply by Y-0930** [reply]
> 再过一段时间有up为了恰饭出手把手教程后，估计环境就差了

**Reply by Erdbronie** [reply]
出手把手教程也有人不会的（

**Reply by Y-0930** [reply]
> 出手把手教程也有人不会的（

哈哈哈，确实。

**Comment by lizard315315** [comment]
非常不错

的环境

**Comment by DonQuixote084** [comment]
门槛还是挺高的，一般没有**混进来，还有交出你的咖啡杯
